sys-devel/dev86/dev86-0.16.21-r4.ebuild

```
@@ -0,0 +1,81 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Bruce's C compiler - Simple C compiler to generate 8086 code"
+HOMEPAGE="http://www.debath.co.uk/ https://github.com/lkundrak/dev86"
+SRC_URI="http://v3.sk/~lkundrak/dev86/Dev86src-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+
+RDEPEND="sys-devel/bin86"
+DEPEND="${RDEPEND}
+	dev-util/gperf"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-pic.patch"
+	"${FILESDIR}/${PN}-0.16.19-fortify.patch"
+	"${FILESDIR}/${P}-non-void-return-clang.patch"
+	"${FILESDIR}/${PN}-0.16.21-make.patch"
```

```suggestion
	"${FILESDIR}/${P}-make.patch"
```

(see also prev/next) lines

---

no, do just as I suggested.